vendor/github.com/containerd/containerd/vendor.conf

```diff
@@ -57,22 +58,26 @@ github.com/golang/glog 44145f04b68cf362d9c4df2182967c2275eaefed
 github.com/google/gofuzz 44d81051d367757e1c7c6a5a86423ece9afcf63c
 github.com/hashicorp/errwrap 7554cd9344cec97297fa6649b055a8c98c2a1e55
 github.com/hashicorp/go-multierror ed905158d87462226a13fe39ddf685ea65f1c11f
-github.com/json-iterator/go 1.0.4
-github.com/opencontainers/runtime-tools 6073aff4ac61897f75895123f7e24135204a404d
+github.com/json-iterator/go f2b4162afba35581b6d4a50d3b8f34e33c144682
```

Comment:
hm, this is somewhere between 1.1.3 and 1.1.4; https://github.com/json-iterator/go/compare/f2b4162afba35581b6d4a50d3b8f34e33c144682...1.1.4 - I can open a PR upstream to make it use a tagged release again